### PR TITLE
Added `uzbek` language

### PR DESCRIPTION
**Pull request by mtakhirov**

### CODE REVIEW

**Review by thetakhirov — APPROVED** [review]

**Review by vlade11115 — APPROVED** [review]

File: uz.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+    <div class="dialog-line -you">
+        <div class="content">
+            N ga tushunadiganlar bormi?
+        </div>
+    </div>
+
+    <div class="dialog-line -you">
+        <div class="content">
+            Buyerda kimdir Yandexdan kurs sotib olganmi? Menda matn tahlili bilan oid savollar bor...
```
Comment (vlade11115) [inline uz.html:44]:
```suggestion
            Bu yerda kimdir Yandexdan kurs sotib olganmi? Menda matn tahlili bilan oid savollar bor...
```

**Review by vlade11115 — CHANGES_REQUESTED** [review]
@thetakhirov please add this and we age good to merge.

File: uz.html (hunk 1 above)
Comment (thetakhirov) [reply uz.html]: Ooh... thank you 🙂

File: uz.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    <div class="dialog-line -you">
+        <div class="content">
+            N ga tushunadiganlar bormi?
+        </div>
+    </div>
+
+    <div class="dialog-line -you">
+        <div class="content">
+            Bu yerda kimdir Yandexdan kurs sotib olganmi? Menda matn tahlili bilan oid savollar bor...
```
Comment (ibrohim777775) [inline uz.html:44]:
```suggestion
            Bu yerda kimdir Yandexdan kurs sotib olganmi? Menda matn tahlili bo'yicha savollar bor...
```
@thetakhirov 
These words are not correct.
This method is easy to read and to understand

**Review by ibrohim777775 — CHANGES_REQUESTED** [review]

File: uz.html (hunk 2 above)
Comment (thetakhirov) [reply uz.html:44]: Thanks 😺

**Review by ibrohim777775 — APPROVED** [review]

**Review by vlade11115 — APPROVED** [review]
@leshark we are good to go.

**Review by vlade11115 — CHANGES_REQUESTED** [review]
@thetakhirov please add uz to language redirect JS code here https://github.com/leshark/nometa/pull/13/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R20
index.html lines 17-21
It detects browser language and redirects to correct translation by default.
I forgot to mention it in the first review.

**Comment by thetakhirov** [comment]
> @thetakhirov please add uz to language redirect JS code here https://github.com/leshark/nometa/pull/13/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R20 index.html lines 17-21 It detects browser language and redirects to correct translation by default. I forgot to mention it in the first review.

@vlade11115 added

**Review by vlade11115 — APPROVED** [review]

**Comment by vlade11115** [comment]
https://nometa.xyz/uz.html
Kudos to @thetakhirov

**Comment by vodiylik** [comment]
@tkhrv @ibrohim777775 nima uchun olingan kontent manbaasiga link berilmagan?
yoki shunchaki o'zbek tili versiyasi uchun nometa.uz saytiga link qo'yib qo'yilavermagan?

**Comment by ibrohim777775** [comment]
> @tkhrv @ibrohim777775 nima uchun olingan kontent manbaasiga link berilmagan? yoki shunchaki o'zbek tili versiyasi uchun nometa.uz saytiga link qo'yib qo'yilavermagan?

qaysi kontent olingan? Bu mana shu saytdagi matnlarni tarjimasi holos.

**Comment by vodiylik** [comment]
> > @tkhrv @ibrohim777775 nima uchun olingan kontent manbaasiga link berilmagan? yoki shunchaki o'zbek tili versiyasi uchun nometa.uz saytiga link qo'yib qo'yilavermagan?
> 
> qaysi kontent olingan? Bu mana shu saytdagi matnlarni tarjimasi holos.

[bad-good.jpg](https://nometa.uz/img/bad-good.png) ga o'xshar ekan.
Hozir qaradim. Ikki xil ekan. Uzr. My bad.

**Pull request merged**